reto.exs

```diff
@@ -0,0 +1,20 @@
+defmodule SeriesUtils do
+  def count_numbers_series(msg, amount, cont, lower_limit, upper_limit) when cont <= upper_limit do
+    msg = msg <> "#{cont}"
+    number = String.to_integer(msg)
+    if cont >= lower_limit do
```

Comment:
Could we do the same as before with this condition?, maybe we can make only one call of `count_numbers_series` `IO.puts(number)` would not be necessary :)

Reply:
@kamala1606 after fixing it you can merge this PR :)